src/core/avis/interface.ts

```diff
@@ -0,0 +1,53 @@
+export interface AvisInfos {
+  name: string;
+  comment: string;
+  evaluation: Evaluation | undefined;
+}
+
+enum Evaluation {
+  BAD_EVALUATION,
+  MEDIUM_EVALUATION,
+  GOOD_EVALUATION,
+}
```

Comment:
```suggestion
enum Evaluation {
  BAD_EVALUATION = "BAD_EVALUATION",
  MEDIUM_EVALUATION,
  GOOD_EVALUATION,
}
```

Pareil pour les autres